src/windows/GitLab.UI.Windows/Program.cs

```diff
@@ -1,5 +1,6 @@
 ﻿using System;
 using System.Diagnostics;
+using System.Threading;
```

Comment:
This `using` import is no longer needed (along with the other UI helper `Program.cs`s.

Reply:
I think the UI Helpers do need it since they name the thread they spawn `AppMain`, right?

Reply:
Ah, well the Linux/macOS ones do but Windows don't. Got it.